sns_ik_lib/src/sns_ik.cpp

```diff
@@ -230,24 +229,33 @@ int SNS_IK::CartToJnt(const KDL::JntArray &q_init, const KDL::Frame &p_in,
     return -1;
   }
 
+  // The position solver uses a barrier function instead of the hard position limits
```

Comment:
Maybe in the future. For now, the solvers probably wouldn't work well set the other way.